build-caching/src/main/java/com/vertispan/j2cl/build/PropertyTrackingConfig.java

```diff
@@ -141,6 +141,15 @@ public String getDependencyMode() {
         return Collections.emptySet();
     }
 
+    @Override
+    public Optional<String> getTranslationsFile() {
```

Comment:
Please return a File instead of a String, and pass it through useFileConfig so that it is cached. Note that using a string in this way and not calling useStringConfig means that even if the string property value changed, the build would not re-run as it should, but in not using a file, if the file contents changed, the build still won't re-run.